spec/spec_helper_acceptance_local.rb

```diff
@@ -72,8 +73,8 @@ def configure_puppet_server(controller, worker1, worker2)
   site_pp = <<-EOS
   node /#{controller[0]}/ {
     class {'kubernetes':
-      kubernetes_version => '1.20.6',
-      kubernetes_package_version => '1.20.6',
+      kubernetes_version => '1.28.15',
```

Comment:
1.28 is already EOL, maybe bump to `1.29` or `1.30`?

Reply:
There are still some work left to fix Debian related issue, I will take this up while fixing it.